src/test/run-make/issue-96498/Makefile

```diff
@@ -0,0 +1,8 @@
+# only-windows-msvc
```

Comment:
```suggestion
# only-windows
```
There seems to be no reasons not to run it on windows-gnu.
Or even on all targets, but it may be more risky because I'm not sure that all targets on CI support cdylibs.